src/styles/_catalog.scss

```
@@ -1,22 +1,30 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  grid-gap: 48px;
 
   max-width: 944px;
   margin: 0 auto;
 
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
```

```suggestion
    grid-template-columns: repeat(2, 1fr);
```
or use variable `$card-width`